3561/3/controllers/comments_controller.rb

```diff
@@ -0,0 +1,12 @@
+require_relative '../controllers/application_controller'
+
+class CommentsController < ApplicationController
+  post '/comments' do
+    Comment.create(text: params[:comment],
+                   star: params[:star],
+                   place_id: params[:id],
+                   user_id: params[:user_id])
+    Place.update_place(Place.find(params[:id]))
```

Comment:
Write method for object of this class, it should call like this, `Place.find(params[:id]).update_rating`